src/control/server/server.go

```diff
@@ -124,14 +132,14 @@ func Start(log *logging.LeveledLogger, cfg *Configuration) error {
 	}
 
 	// Create and setup control service.
-	controlService, err := NewControlService(log, harness, scmProvider, cfg, membership)
+	controlService, err := NewControlService(log, harness, bdevProvider, scmProvider, cfg, membership)
 	if err != nil {
 		return errors.Wrap(err, "init control service")
 	}
 	if err := controlService.Setup(); err != nil {
 		return errors.Wrap(err, "setup control service")
 	}
-	defer controlService.Teardown()
+	//defer controlService.Teardown()
```

Comment:
NB: will remove this in a subsequent commit, assuming this will take at least another iteration before landing